services/gateway/apis/http-version3/swagger/parameters/tokens.json

```diff
@@ -5,14 +5,29 @@
         "description": "Token ID to query",
         "type": "string",
         "format": "id",
-        "minLength": 8,
-        "maxLength": 8
+        "minLength": 16,
+        "maxLength": 16
+    },
+    "tokenIDCSV": {
+        "name": "tokenID",
+        "in": "query",
+        "description": "Token ID to query (can be expressed as CSV)",
+        "type": "string",
+        "format": "id",
+        "minLength": 16
     },
     "tokenName": {
         "name": "tokenName",
         "in": "query",
         "description": "Token Name to query",
         "type": "string",
         "minLength": 1
+    },
+    "tokenNameCSV": {
+        "name": "tokenName",
+        "in": "query",
+        "description": "Token Name to query (can be expressed as CSV)",
+        "type": "string",
+        "minLength": 1
```

Comment:
Same as above.

Reply:
for tokenName we will do. `tokenID` is also being used for `/tokens` endpoint where it is not accepted as CSV

Reply:
Applied.